packages/workspace/src/utils/buildable-libs-utils.ts

```diff
@@ -179,11 +179,19 @@ export function checkDependentProjectsHaveBeenBuilt(
 
 export function updatePaths(
   dependencies: DependentBuildableProjectNode[],
-  paths: { [k: string]: string[] }
+  paths: Record<string, string[]>
 ) {
+  const deps = Object.keys(paths);
```

Comment:
can you call it pathsKeys?

Cause you already have dependencies as an argument, and then you do `(const path of deps)`.

Reply:
ok, np